### DISCUSSION TITLE
G7 working for 3 months, no connecting to new sensor now.

### DISCUSSION THREAD

**Discussion by RayDubbs** [NightscoutFoundation/xDrip#3708] [Q&A]
Hi everyone, I'm running G7 on android (Samsung).  It's been working fine for 3 months but now I can't seem to connect to a new sensor I just started.   I've tried everything in the suggested troubleshooting, any help would be amazing.  Thank you. 
![Screenshot_20241010_092201_xDrip+](https://github.com/user-attachments/assets/34895a3b-33e1-46c6-a3e5-1d27824bf7b8)
![Screenshot_20241010_092143_xDrip+](https://github.com/user-attachments/assets/edabe62b-a9a4-4aa6-bf8e-8fb86fc8611f)

**Comment by Navid200** [comment]
Can you confirm that you have inserted the sensor?
And you have nothing else connected to it?

If the answer to both is yes, can you try a scanner?
https://navid200.github.io/xDrip/docs/Bluetooth-Scanner.html

**Reply by RayDubbs** [reply]
Thank you for the reply.  I can confirm both.  I'm getting the scanner app and giving that a try.

**Reply by RayDubbs** [reply]
Used the scanner.  It's not showing up.  I wonder if I have a bad sensor out of the box?

**Reply by Navid200** [reply]
Can you take the applicator and bring it close to the sensor such the magnet in it comes close to the sensor for a few seconds?
Then, wait 5 minutes.  Does it show a pair request?  You may need to raise the volume so that you can hear a pair request.

**Reply by RayDubbs** [reply]
I don't have the applicator with me but I do have magnets.  Can I use a regular magnet or does it have to be with the applicator?

**Reply by Navid200** [reply]
I suppose you can use a magnet.  There is a mechanism inside the sensor that is supposed to start working the moment the sensor is applied as it moves away from the magnet.
There are reports of people having experienced a sensor not working and bringing the magnet close for a few seconds has triggered it successfully to start.

**Reply by RayDubbs** [reply]
I'll give it a shot with what I have here.

**Reply by RayDubbs** [reply]
Refrigerator magnet next to the sensor for about 15 seconds, removed the magnet then the sensor woke up and is now warming up.  Thank you so much for your help!!

**Reply by Navid200** [reply]
Thanks for the feedback.

Thanks go to @tcbh for reporting this and teaching me this and to the person who originally reported it in German in the forum that @tcbh shared with us.  

I am adding this to the guide.

**Reply by Navid200** [reply]
https://github.com/NightscoutFoundation/xDrip/discussions/3706
